folly/Traits.h

```diff
@@ -384,10 +384,24 @@ struct Bools {
   }
 };
 
+template <bool... Bs>
+struct AllFalse : std::is_same<Bools<Bs...>, Bools<(Bs && false)...>> {};
+
+template <bool... Bs>
+struct AllTrue : std::is_same<Bools<Bs...>, Bools<(Bs || true)...>> {};
```

Comment:
Besides that I'll remove `AllTrue`, `AllFalse` etc. IMHO they're named well. Names `AllOf`, `AnyOf` etc. suggests that they take some kind predicate and check all types against it (like in the STL), don't you think?

>I prefer to keep the API small and simple. 
That was my problem. I thought about folly as a library that need to provide as wide API as possible. Now I understand that new stuff shall be added when there is a need.